modules/core/core-api/src/main/java/com/enonic/xp/api/ApiDescriptor.java

```diff
@@ -15,12 +17,24 @@ public final class ApiDescriptor
 
     private final PrincipalKeys allowedPrincipals;
 
+    private final String displayName;
+
+    private final String description;
+
+    private final String documentationUrl;
+
+    private final Boolean useInSlashApi;
```

Comment:
i
```suggestion
    private final boolean useInSlashApi;
```

if it is always set, why making it nullable?